docs/getting_started/access_account.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Accessing your Custodian account
```

Comment:
```suggestion
title: Accessing Custodian accounts
```
The terminology here is extremely ungrateful to work with.
Custodian is the entity which holds the keys.
Under security/custodian, we're talking about accounts for which we have keys in the KMS (the vault).

On the other (linguistical) hand, a custodian account would be an account who has keys in custody.
That's not the case here.

So these accounts are the ones for which Calimero is the custodian.
Extremely ungrateful to work with from the language perspective.

According to chat GPT, we could use the term "Custodial accounts".
This conveys that we are in custody of the accounts.
Custodian is wrongly used here.

Reply:
Naturally, this needs to be changed in the console as well.
https://calimero.atlassian.net/browse/AM-1173